nan_persistent_12_inl.h

```diff
@@ -81,6 +87,8 @@ class Global : public v8::Global<T> {
     v8::PersistentBase<T>::Reset(v8::Isolate::GetCurrent(), other);
   }
 
+  Global Pass() { return static_cast<Global&&>(*this); }  // NOLINT(build/c++11)
```

Comment:
Shouldn't that be in an `#if __cplusplus >= 201103L` to avoid breaking g++ 4.7 and older?  v0.10 and v0.12 support g++ 4.2.